docs/guide/dataset_pretrained_models.md

```diff
@@ -0,0 +1,24 @@
+# Dataset and pre-trained models
+
+The purpose of providing a sample dataset and pre-trained models here is to help beginners to get started easier. You can download the dataset and use it to train together with your own dataset. 
+
+
+## Dataset
+
+### Dataset in real world
+
+1. Sample dataset #1 by Tawn - 2.0GB , [Link](https://drive.google.com/file/d/13dXSoHx6YpTvesdyUMe6o_ZfyYTYlfaK/view?usp=sharing)
```

Comment:
Link to Tawn's user profile in GitHub - that way people know who you are referring to.